content/webapp/pages/stories/[articleId].tsx

```diff
@@ -194,7 +225,51 @@ const ArticlePage: FunctionComponent<Props> = ({ article, jsonLd }) => {
       }
     }
 
+    async function getRelatedDoc() {
+      if (article.exploreMoreDocument?.type === 'exhibitions') {
+        const relatedExhibition =
+          await fetchFromClientSide<RawExhibitionsDocument>({
+            endpoint: 'exhibitions/exhibition',
+            params: {
+              id:
+                article.exploreMoreDocument?.uid ||
+                article.exploreMoreDocument?.id,
+            },
+          });
+
+        if (relatedExhibition) {
+          // We're hiding exhibitions once they've ended, so not setting the related document value.
+          const isPastExhibition = isPast(
+            transformTimestamp(relatedExhibition.data.end) || new Date()
```

Comment:
just checking, we want this to be `false` if there isn't a `relatedExhibition.data.end`?

Reply:
I think it'd make sense right? It would mean permanent, so never `isPast`?